latexify/core.py

```diff
@@ -75,8 +75,38 @@ def visit_Call(self, node):  # pylint: disable=invalid-name
       rstr = r'\right)'
 
     arg_strs = [self.visit(arg) for arg in node.args]
+
+    if callee_str == 'sum' and isinstance(node.args[0], ast.GeneratorExp):
+      limit_str, formula_str = self.visit(node.args[0])
+      lstr =  r'\sum' + limit_str + r' \left({'
+      return lstr + formula_str + rstr
+    if callee_str == 'range':
+      return arg_strs
+
     return lstr + ', '.join(arg_strs) + rstr
 
+  def visit_GeneratorExp(self, node):  # pylint: disable=invalid-name
+    limit_str = self.visit(node.generators[0])
```

Comment:
These variable names seems to be specialized for limit-like operators. General naming is better. (ditto for others)